src/main/java/com/example/springboot/dao/BookDAOService.java

```diff
@@ -0,0 +1,162 @@
+package com.example.springboot.dao;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+import org.apache.log4j.Logger;
+import org.springframework.stereotype.Repository;
+
+import com.example.springboot.entity.Book;
+import com.mysql.jdbc.Statement;
+
+/**
+ * This class implement methods from IBookDAOService interface
+ * @author Kohut Dmytro
+ * @version 1.0
+ */
+@Repository("iBookService")
+public class BookDAOService implements IBookDAOService {
+
+	private final Logger LOG = Logger.getLogger(BookDAOService.class);
```

Comment:
we have a standard where all loggers should be `private final static` and if it is static should be written with UPPERCASE `LOGGER`